composer.json

```diff
@@ -65,10 +68,21 @@
         ],
         "post-install-cmd": [
             "DrupalProject\\composer\\ScriptHandler::createRequiredFiles",
-            "@drupal-scaffold"
+            "@drupal-scaffold",
+            "@kalastatic"
```

Comment:
KalaStatic may not be installed on every project, but this does seem useful. Let's keep 'er in! :+1: